Task.123/Fqdn.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+import email
+import os
+import requests
+import socket
+def main():
```

Comment:
please fix indentation in your main() 7-69 listing
should be either 2 or 4 whitespaces